pkg/controller/tidbcluster/tidb_cluster_control.go

```diff
@@ -219,7 +219,9 @@ func (c *defaultTidbClusterControl) updateTidbCluster(tc *v1alpha1.TidbCluster)
 		return err
 	}
 
+	// works that should be done to make the ticdc cluster current state match the desired state:
```

Comment:
Do you need to sync CDC before TiDB?

Reply:
No need, CDC only relies on TiKV and PD.